File: src/2022/day04/src/main/java/ca/terrylockett/aoc2022/day04/Day04Runner.java
package ca.terrylockett.aoc2022.day04;

import ca.terrylockett.aoccommon.inputfilefinder.Resources;

public class Day04Runner {

	public static void main(String[] args) throws Exception {
		String inputFile = Resources.getInputFilePath("input.txt").orElseThrow();

		// ***** part 01 *****
		int count = CampAssignmentCleaner.findAssignmentsErrorsCount(inputFile);
		System.out.println("Part 1: " + count);

		// ***** part 01 *****
		count = CampAssignmentCleaner.findAssignmentsErrorsCount(inputFile, true);
		System.out.println("Part 2: " + count);

	}

}

File: src/2023/day04/src/main/kotlin/ca/terrylockett/aoc2023/day04/Day04Runner.kt
package ca.terrylockett.aoc2023.day04

import ca.terrylockett.aoccommon.inputfilefinder.Resources

fun main() {
	val inputFile: String = Resources.getInputFilePath("input.txt").orElseThrow()

	println("day04 2023 part1: ${part1GetPoints(inputFile)}")
	println("day04 2023 part2: ${part2GetPoints(inputFile)}")
}
